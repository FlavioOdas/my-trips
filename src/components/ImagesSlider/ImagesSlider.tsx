import React from 'react'
import Slider, { Settings } from 'react-slick'

const ImagesSlider = (props: Settings) => {
  const defaultSettings: Settings = {
    dots: false,
    infinite: true,
    speed: 500,
    slidesToShow: 1,
    slidesToScroll: 1
  }

  const settings = { ...defaultSettings, props }

  return (
    <Slider {...settings}>
      <div>
        <img
          src="https://i.redd.it/421tpw4mtnu91.png"
          width="100%"
          height={350}
          alt="Hunting trip"
        />
      </div>
      <div>
        <img
          src="https://i.redd.it/8xm7w23xymv91.jpg"
          width="100%"
          height={350}
          alt="Yesterday's sunset on I-84 in CT"
        />
      </div>
      <div>
        <img
          src="https://i.redd.it/864sx9p4x3v91.png"
          width="100%"
          height={350}
          alt="Orange Cars"
        />
      </div>
    </Slider>
  )
}

export default ImagesSlider
