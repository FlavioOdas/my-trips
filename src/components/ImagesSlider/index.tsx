import ImagesSlider from './ImagesSlider'

export default ImagesSlider
