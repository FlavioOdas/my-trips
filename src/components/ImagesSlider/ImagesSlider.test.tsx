import { render } from '@testing-library/react'
import React from 'react'

import ImagesSlider from './ImagesSlider'

describe('ImagesSlider', () => {
  it('should render the component', () => {
    const component = render(<ImagesSlider />)

    expect(component.asFragment).toMatchSnapshot()
  })
})
