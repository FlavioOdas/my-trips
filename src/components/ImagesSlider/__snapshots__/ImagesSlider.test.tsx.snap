// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagesSlider should render the component 1`] = `[Function]`
